src/main/java/lotto/model/BonusNumber.java

```diff
@@ -0,0 +1,36 @@
+package lotto.model;
+
+import java.util.List;
+import lotto.util.ExceptionHelper;
+
+import static lotto.model.Lotto.DUPLICATE_NUMBER_ERROR_MESSAGE;
+import static lotto.model.constant.Lotto.MAX_NUMBER;
+import static lotto.model.constant.Lotto.MIN_NUMBER;
+
+public class BonusNumber {
+    public final static String INVALID_NUMBER_RANGE_ERROR_MESSAGE = "번호는 1 이상 45 이하로 입력해주세요.";
+
+    private final int bonus;
+
+    public BonusNumber(final List<Integer> numbers, final int bonus) {
+        validateNumberRange(bonus);
+        validateDuplicate(numbers, bonus);
+        this.bonus = bonus;
+    }
+
+    private void validateDuplicate(final List<Integer> numbers, final int bonus) {
+        if (numbers.contains(bonus)) {
+            throw new IllegalArgumentException(ExceptionHelper.errorMessage(DUPLICATE_NUMBER_ERROR_MESSAGE));
+        }
+    }
+
+    private void validateNumberRange(final int bonus) {
+        if (!(MIN_NUMBER <= bonus && bonus <= MAX_NUMBER)) {
```

Comment:
부정연산자는 코드를 읽는 입장에서 한번 더 생각해야하니 부정연산자를 없애고 다음과 같이 바꿔주면 어떨까요? 괄호안에 낑겨있어서 잘 안보이기도 하고요!
```suggestion
        if (bonus < MIN_NUMBER || bonus > MAX_NUMBER) {
```

Reply:
네 확실히 부정연산자보다 더 가독성이 좋군요!
다음 미션에서 적용해보겠습니다.